src/pybind/mgr/cephadm/module.py

```diff
@@ -2163,7 +2163,7 @@ def _get_container_image_id(self, image_name):
         if not host:
             raise OrchestratorError('no hosts defined')
         out, err, code = self._run_cephadm(
-            host, None, 'pull', [],
+            host, '', 'pull', [],
```

Comment:
unfortunately, the upgrade test totally failed: http://pulpito.ceph.com/swagner-2020-06-23_11:55:14-rados:cephadm-wip-swagner-testing-2020-06-23-1057-distro-basic-smithi/5172323/